src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2310,6 +2332,36 @@ public async Task GetAsync_CallMethod_ExpectedStatusLine(HttpStatusCode statusCo
                 $"HTTP/1.1 {(int)statusCode} {reasonPhrase}\r\nContent-Length: 0\r\n\r\n"));
         }
 
+        [Fact]
+        public async Task PostAsync_CustomTransferEncoding()
+        {
+            string data = "Test String";
```

Comment:
```
const string TestContent = "Test String";
```

Also, in this test you've defined it in a const to be shared across all uses, but in the previous test it's not... it'd be good to change the other test to also use one.

Reply:
Sure, added that to both tests.